src/navigators/HomeStack.tsx

```diff
@@ -8,14 +8,14 @@ import { TrackTileTrackerScreen } from '../screens/TrackTileTrackerScreen';
 import { AppNavHeader } from '../components/AppNavHeader';
 import { TrackerValuesContext } from '../components/TrackTile/main';
 import TrackTileSettingsScreen from '../screens/TrackTileSettingsScreen';
-import { t } from '@i18n';
+import { t } from '../../lib/i18n';
 
 export type HomeStackParamList = {
   Home: undefined;
-  'tiles/AppTile': { appTile: AppTile };
-  'tiles/CustomAppTile': { appTile: AppTile };
-  'tiles/TrackTile': { tracker: any; valuesContext: any };
-  'tiles/trackTileSettings': {
+  'Home/AppTile': { appTile: AppTile };
+  'Home/CustomAppTile': { appTile: AppTile };
+  'Home/TrackTile': { tracker: any; valuesContext: any };
+  'Home/TrackTileSettings': {
```

Comment:
Initial gut reaction is I didn't think I liked this - thought these belong "to the TilesList".  But indeed, the `TilesList` could be used across several stacks.. so the prefix should represent (if anything) the stack it's in.  I could see an argument for not having the prefix at all, but if `/TrackTile` were also in the Settings stack, for example, these prefixes allow us to be specific about which one we want to navigate to.  I dig it!